src/logs/logger.ts

```diff
@@ -0,0 +1,38 @@
+import winston, { transports } from 'winston';
+// import * as fs from 'node:fs';
+// import path from 'node:path';
+
+// const logDir = '../../src/logs';
+
+// if (!fs.existsSync(logDir)) {
+//   fs.mkdirSync(logDir);
+// }
+
+const { combine, timestamp, json } = winston.format;
+const { printf, colorize, align } = winston.format;
+
+winston.loggers.add('basicLogger', {
+  level: process.env.LOG_LEVEL || 'info',
```

Comment:
Add `LOG_LEVEL` to .env.example and set it to debug. During development, it's good to set log level to debug

Reply:
Noted sir, but sir, I have questions about the logger